lightning/src/onion_message/messenger.rs

```diff
@@ -954,7 +1015,18 @@ where
 					},
 				}
 			},
-			Ok(PeeledOnion::Forward(next_node_id, onion_message)) => {
+			Ok(PeeledOnion::Forward(next_hop, onion_message)) => {
+				let next_node_id = match next_hop {
+					NextHop::NodeId(pubkey) => pubkey,
+					NextHop::ShortChannelId(scid) => match self.node_id_lookup.next_node_id(scid) {
+						Some(pubkey) => pubkey,
+						None => {
+							log_trace!(self.logger, "Dropping forwarded onion message to unresolved peer using SCID {}", scid);
```

Comment:
nit: Maybe we can rephrase it a little.

```suggestion
							log_trace!(self.logger, "Dropping forwarded onion message: Unable to resolve next peer using SCID {}", scid);
```

Reply:
Done.